rpcs3/stdafx.h

```diff
@@ -212,3 +212,7 @@ template<typename T> struct id_traits;
 #include "Utilities/BitField.h"
 #include "Utilities/File.h"
 #include "Utilities/Log.h"
+
+
+#define MAX2(a, b) (((a) > (b)) ? (a) : (b))
+#define MIN2(a, b) (((a) > (b)) ? (b) : (a))
```

Comment:
What is the purpose of these macro? Why not replace them with std::min/max in the first place?